tox.ini

```diff
@@ -7,7 +7,7 @@ envlist = flake8,ansible-lint
 basepython = python2.7
 deps =
     ansible>=1.9.1,<2.0.0
-    -ropenstack-ansible/test-requirements.txt
+    -rtest-requirements.txt
```

Comment:
I see openstack-ansible doesn't actually have a test-requirements file in Kilo, but with the exception of sphinx libraries, I'm not we're saving much by carrying a separate test-requirements.txt file. Was there a specific reason for doing so?